README.md

```diff
@@ -25,17 +25,12 @@ This package provides a KeyShot plugin script that creates jobs for AWS Deadline
 
 ### Using the KeyShot Submitter
 
-1. Install deadline and Pyside6
-    - e.g. `pip install deadline[gui]`
-2. Copy or link the file `deadline-cloud-for-keyshot/keyshot_script/Submit to AWS Deadline Cloud.py` to the KeyShot scripts folder.
-    - e.g. On Windows `C:/Users/<USER>/Documents/KeyShot/Scripts`
-3. Set the following environment variables
-    - Set the environment variable `DEADLINE_PYTHON` as the path to the Python installation where deadline-cloud and PySide2 were installed in step 1.
-      - e.g. On Windows if using Python 3.10 it might be `set DEADLINE_PYTHON=C:/Users/<USER>/AppData/Local/Programs/Python/Python310/python`
-    - Set the environment variable `DEADLINE_KEYSHOT` as the path to the `<PATH TO>/deadline-cloud-for-keyshot/src/deadline/keyshot_submitter` folder
-      - e.g. On Windows if the source was on the user's desktop it might be  `set DEADLINE_KEYSHOT=C:/Users/<USER>/Desktop/deadline-cloud-for-keyshot/src/deadline/keyshot_submitter`
-4. Launch KeyShot with the environment variables from step 3. set.
-5. The submitter can be launched within KeyShot from `Window > Scripting Console > Scripts > Submit to AWS Deadline Cloud > Run`
+1. Install `deadline[gui] >= 0.48.7,< 0.49` and `PySide6`
+    - e.g. `pip install "deadline[gui] >= 0.48.7,< 0.49" PySide6`
```

Comment:
why specify PySide6 in the command? `deadline[gui]` already brings the version of `Pyside6-Essentials` that it requires. Whereas just PySide6 will bring a lot of additional addons

Reply:
Removed PySide6 from the `pip install` command. Is `pip install deadline-cloud-for-keyshot` necessary? If someone is running these instructions, I understand that they would be doing some sort of local development and having an install of `deadline-cloud-for-keyshot` may cause more confusion around import/path priority? If someone doesn't want to do development of the KeyShot submitter, I understand that they would just install it from the Deadline Cloud binaries following our public docs. We could mention that here as well once we release the latest binaries.

Reply:
Gotcha. In my mind this was related to the manual installation instructions, rather than the development instructions. Future improvement we can punt.

Reply:
> Is pip install deadline-cloud-for-keyshot necessary?

It depends on what the instructions are targeting. Doing this gets you the correct version of deadline which will then mean we don't have to keep updating these instructions on new releases.